api-service/job-server-demo/README.md

```diff
@@ -23,28 +23,47 @@ Then from this directory, use yarn to install the local package dependencies:
 yarn install
 ```
 
-Link in the Wikipedia conversations library (to treat it as a node package):
+Setup the initial `build` directory.
 
-```shell
-cd node_modules
-ln -s ../../wpconvlib ./
-cd ..
+```
+yarn run setup
 ```
 
-### Setup the config file
+### Setup the config file and static webapp
 
 Before you can deploy, you need to:
 
-1. Copy the `server_config.template.json` file to `build/config/server_config.json`.
-2. In the `build/config/server_config.json` file, set these values:
+1.  Copy the `server_config.template.json` file to `build/config/server_config.json`.
```

Comment:
Maybe add that yarn run setup will do this for you. :)

Reply:
This one I think we shouldn't do because as there becomes new front-ends, it would be the wrong thing most of the time.